report_thesis/src/sections/problem_definition.tex

```diff
@@ -20,71 +32,97 @@ \section{Problem Definition}\label{sec:problem_definition}
 
-The metric we use to evaluate the accuracy of our models is the \gls{rmse}. \gls{rmse} is calculated using the formula:
+The metric used to evaluate the accuracy of the models is the Root Mean Squared Error (RMSE):
 \[
 \text{RMSE} = \sqrt{\frac{1}{n} \sum_{i=1}^{n} (\mathbf{v}_i - \hat{\mathbf{v}}_i)^2}
 \]
-where \( \mathbf{v}_i \) is the vector of actual oxide concentrations for the \( i \)-th sample, \( \hat{\mathbf{v}}_i \) is the corresponding vector of predicted oxide concentrations, and \( n \) is the total number of samples. This measure quantifies the average magnitude of the prediction error across all predicted values, providing a clear indication of model accuracy in quantifying chemical compositions.
+where \( \mathbf{v}_i \) is the vector of actual oxide concentrations for the \( i \)-th sample, \( \hat{\mathbf{v}}_i \) is the corresponding vector of predicted oxide concentrations, and \( n \) is the total number of samples. This measure quantifies the average magnitude of the prediction error across all predicted values.
 
-We evaluate the robustness of our models using the sample standard deviation of prediction errors, defined as:
+Robustness is evaluated using the sample standard deviation of prediction errors:
 \[
 \sigma_{error} = \sqrt{\frac{1}{n-1} \sum_{i=1}^{n} (e_i - \bar{e})^2}
 \]
 where \( e_i = \mathbf{v}_i - \hat{\mathbf{v}}_i \) and \( \bar{e} \) is the mean error.
-Sample standard deviation is used because it provides an unbiased estimate of the variability in prediction errors, crucial for assessing how well the model can be expected to perform on new, unseen data.
-Correcting the variance calculation by using \( n-1 \) instead of \( n \) compensates for the tendency of smaller samples (specific datasets of \gls{libs} spectral data) to underestimate the variability of the entire population (all possible \gls{libs} data), bringing the sample standard deviation closer to the true standard deviation of the entire population.
 A lower standard deviation indicates a more robust model across different samples.
 
-Success will be evaluated primarily through these metrics, comparing the predictive accuracy and robustness of our models against existing benchmarks and baseline models established in prior research.
+\subsection{Challenges}
+As mentioned, quantifying chemical compositions from \gls{libs} spectral data involves several significant challenges that must be addressed to ensure accurate and robust predictions.
+Several of the following challenges can be addressed through known methodologies, such as sample preprocessing, ensuring the repeatability of observations, and selecting calibration samples that are matrix-matched to specific targets of interest\cite{andersonPostlandingMajorElement2022}.
+However, these strategies are not feasible for the extraterrestrial applications relevant to this research.
```

Comment:
Why? Might need to elaborate

Reply:
They don't exactly spell it out in the paper (see img), but likely because we don't have the actual, physical samples here on Earth. Want me to clarify that? Wanted to avoid doing so by referring to the paper that made the claim originally

![image](https://github.com/chhoumann/thesis-chemcam/assets/29108628/b065e7e8-28a1-4f0c-8067-6efe19dc01bb)